packages/react-server/src/ReactFlightServer.js

```diff
@@ -184,10 +184,10 @@ function devirtualizeURL(url: string): string {
     // We need to reverse it back into the original location by stripping its prefix
     // and suffix. We don't need the environment name because it's available on the
     // parent object that will contain the stack.
-    const envIdx = url.indexOf('/', 12);
+    const envIdx = url.indexOf('/', 'rsc://React/'.length);
```

Comment:
Does this get optimized by closure?

Reply:
Yep: https://github.com/facebook/react/pull/33392#discussion_r2126423839